d2bs/kolbot/D2BotMain.dbj

```diff
@@ -0,0 +1,298 @@
+/**
+ * @description Just the regular bot you want to select, whenever you want to join a game or host a game
+ * @author Jaenster
+ */
+const StarterConfig = {
+	MinGameTime: 120, // Minimum game length in seconds. If a game is ended too soon, the rest of the time is waited in the lobby
+	PingQuitDelay: 30, // Time in seconds to wait in lobby after quitting due to high ping
+	CreateGameDelay: 10, // Seconds to wait before creating a new game
+	ResetCount: 999, // Reset game count back to 1 every X games.
+	CharacterDifference: 99, // Character level difference. Set to false to disable character difference.
+	ChatActionsDelay: 2, // Seconds to wait in lobby before entering a channel
+
+	// ChannelConfig can override these options for individual profiles.
+	JoinChannel: "", // Default channel. Can be an array of channels - ["channel 1", "channel 2"]
+	FirstJoinMessage: "", // Default join message. Can be an array of messages
+	AnnounceGames: false, // Default value
+	AfterGameMessage: "", // Default message after a finished game. Can be an array of messages
+
+	SwitchKeyDelay: 5, // Seconds to wait before switching a used/banned key or after realm down
+	CrashDelay: 5, // Seconds to wait after a d2 window crash
+	FTJDelay: 10, // Seconds to wait after failing to create a game
+	RealmDownDelay: 3, // Minutes to wait after getting Realm Down message
+	UnableToConnectDelay: 5, // Minutes to wait after Unable To Connect message
+	CDKeyInUseDelay: 5, // Minutes to wait before connecting again if CD-Key is in use.
+	ConnectingTimeout: 20, // Seconds to wait before cancelling the 'Connecting...' screen
+	PleaseWaitTimeout: 10, // Seconds to wait before cancelling the 'Please Wait...' screen
+	WaitInLineTimeout: 60, // Seconds to wait before cancelling the 'Waiting in Line...' screen
+	GameDoesNotExistTimeout: 30 // Seconds to wait before cancelling the 'Game does not exist.' screen
+}, ChannelConfig = {};
+
+include('require.js');
+
+function main() {
+	let status = '', fromTickStatus, firstLogin = true, loginRetry = 0;
+
+	// Take care of the HeartBeat
+	const HeartBeat = require('HeartBeat');
+	while (!(HeartBeat.handle && Object.keys(HeartBeat.gameInfo).length)) delay(10);
+
+	// If you press R, it restarts the script, for developing very handy
+	require('Debug').restarter(82); // Press R to restart the script
+
+	include("OOG.js") && include('common/Misc.js') && include('sdk.js') && include('polyfill.js') && include('common/prototypes.js');
+	if (!FileTools.exists("data/" + me.profile + ".json")) DataFile.create();
+	let nextGame = DataFile.getStats()['nextGame'];
```

Comment:
We should set up some linting to standardize when to use `let/const` and other code smells.